src/language/methods.md

```diff
@@ -135,7 +135,7 @@ wrapping them with methods, all without changing client code.
 
 Instance, getter, and setter methods can be abstract, defining an
 interface but leaving its implementation up to other classes.
-Abstract methods can only exist in [abstract classes][].
+Abstract methods can only exist in [abstract classes][] or [mixins][].
 
 To make a method abstract, use a semicolon (;) instead of a method body:
 
```

Comment:
May be useful to add a simple code example of an abstract getter in a mixin, but maybe that could be left to the mixins docs.

Reply:
Good idea. I think you're right though that it should be on the mixin page. It can potentially be added as part of https://github.com/dart-lang/site-www/issues/4871 :)